cartofacade/testhelpers.go

```diff
@@ -32,17 +32,17 @@ func GetBadTestConfig() CartoConfig {
 func GetTestAlgoConfig() CartoAlgoConfig {
 	return CartoAlgoConfig{
 		optimizeOnStart:      false,
-		optimizeEveryNNodes:  0,
-		numRangeData:         0,
-		missingDataRayLength: 0.0,
-		maxRange:             0.0,
-		minRange:             0.0,
-		maxSubmapsToKeep:     0,
-		freshSubmapsCount:    0,
-		minCoveredArea:       0.0,
-		minAddedSubmapsCount: 0,
-		occupiedSpaceWeight:  0.0,
-		translationWeight:    0.0,
-		rotationWeight:       0.0,
+		optimizeEveryNNodes:  3,
```

Comment:
These all being zeroed out was why none of the GetPosition & GetInternalState responses were succeeded in go tests. Cartographer (very unsportingly) never reports errors caused by poor configuration, it simply doesn't do what it should do & leaves it to the user to figure out why they are not getting any outputs for their lidar readings.

This change brings these tests on par with the C++ unit tests.